src/pages/Stream/providers/LogsProvider.tsx

```diff
@@ -705,7 +705,7 @@ const setCleanStoreForStreamChange = (store: LogsStore) => {
 			pinnedColumns: [],
 			filters: {},
 		},
-		...updatedTimeRange,
+		...(timeRange.startTime !== getDefaultTimeRange().startTime ? { timeRange } : updatedTimeRange),
```

Comment:
Review this line

Reply:
We can add more stricter check with endTime and startTime. Any other suggestions?

Reply:
Addressed this.